examples-extra/basic/sdk/src/generatedNoCheck/ontology/queries/queryTakesAllParameterTypes.ts

```diff
@@ -0,0 +1,73 @@
+import { QueryDefinition } from '@osdk/api';
+import { Todo } from '../objects.js';
+export const queryTakesAllParameterTypes = {
+  apiName: 'queryTakesAllParameterTypes',
+  description: 'description of the query that takes all parameter types',
+  displayName: 'qTAPT',
+  type: 'query',
+  version: 'version',
+  parameters: {
+    double: { description: 'a double parameter', type: 'double', nullable: false },
+    float: { type: 'float', nullable: false },
+    integer: { type: 'integer', nullable: false },
+    long: { type: 'long', nullable: false },
+    attachment: { type: 'attachment', nullable: false },
+    boolean: { type: 'boolean', nullable: false },
+    date: { type: 'date', nullable: false },
+    string: { type: 'string', nullable: false },
+    timestamp: { type: 'timestamp', nullable: false },
+    object: {
+      description: 'undefined',
+      type: 'object',
+      object: 'Todo',
+      nullable: false,
+      __OsdkTargetType: Todo,
+    },
+    objectSet: {
+      description: 'undefined',
+      type: 'objectSet',
+      objectSet: 'Todo',
+      nullable: false,
+      __OsdkTargetType: Todo,
```

Comment:
@ericanderson is this not codifying the target type here? I may be misunderstanding your comment

Reply:
Oh I see. Still wouldn't rely on it

Reply:
To give more color, I want the option to remove the runtime cost of these and reduce them to just compile time types (for the most part). In order to do that, we need to not rely on this from bundling but instead to get the latest from the server before execution.